pyproject.toml

```diff
@@ -139,6 +139,7 @@ all_extras = [
     "tsfresh>=0.17.0",
     "tslearn>=0.5.2,<0.6.0; python_version < '3.11'",
     "xarray",
+    "arch>=6.1.0",
```

Comment:
Please also add an upper bound, and the current practice we are following is the next upcoming minor version.

Reply:
Also, please update other extras like all_extras_pandas2 and forecasting